R/catmaid_search.R

```diff
@@ -0,0 +1,53 @@
+# Searching for CATMAID neurons
+
+#' Search for CATMAID skeletons within a volume
+#'
+#' @description  Programmatically search for skeleton IDs pertaining to neurons within a search volume defined by a bounding box.
+#' @param bbx the bounding box (a matrix of 2 rows and 3 columns) describing a search volume
+#' @param min_nodes the minimum number of nodes a neuron in the search area must have (includes nodes outside search area)
+#' @param pid project id. Defaults to 1
+#' @param conn CATMAID connection object, see ?catmaid::catmaid_login for details
+#' @param ... methods passed to catmaid::catmaid_fetch
+#' @export
+#' @rdname catmaid_skeletons_in_bbx
+catmaid_skeletons_in_bbx <- function(bbx, min_nodes = 2, pid = 1, conn = NULL, ...){
```

Comment:
@SridharJagannathan FYI.

Reply:
@jefferis it would be nice if for all these functions there could be `examples`, even if they are configured as `not run`.